packages/components/src/range-control/README.md

```diff
@@ -184,6 +184,14 @@ The maximum value accepted. If higher values are inserted onChange will not be c
 - Type: `Number`
 - Required: No
 
+#### step
```

Comment:
The component don't use a "step" prop explicitely but instead defers all extra props to the inputs (range and number). This raises the question of whether it should be used explicitely in code or if the documentation should mention that any extra prop is applied to both the number and the range inputs.

Reply:
It would be good to mention about additional props being applied in docs. We have seen similar PR / queries.

Reply:
I concur with @ajitbohra . It would be good for the documentation to be as explicit and thorough as possible, so one can easily find what they’re looking for without digging deep into the code.

Reply:
Unless I've misinterpreted, I believe @ajitbohra may specifically be referring to mentions in other components where there's a statement about how additional props are passed through to the underlying input, but still not enumerating the individual props. 

The proposed benefit here is to avoid needing to list and maintain descriptions for [every possible attribute for an input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes_common_to_all_input_types).

Examples:

>MenuItem supports the following props. **Any additional props are passed through to the underlying Button or IconButton component.**

https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/designers/menu-item.md

>The component accepts the following props. **Any other props are passed through to the underlying Popover component (refer to props documentation).**

https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/url-popover/README.md

>NavigableContainer is exported as two classes: NavigableMenu and TabbableContainer. NavigableContainer itself is not exported. NavigableMenu and TabbableContainer have the props listed below. **Any other props will be passed through to the `div`.**

https://github.com/WordPress/gutenberg/blob/master/packages/components/src/navigable-container/README.md

Reply:
Was referring to adding info that users can pass additional props rather than explicitly listing all props.